bower.json

```diff
@@ -31,6 +31,6 @@
     "angular-scenario": ">=1.5.0"
   },
   "resolutions": {
-    "angular": "1.5.0"
+    "angular": ">=1.5.0"
   }
```

Comment:
I'm not sure why angular's resolution was set to a specific version. All the other packages look to be using approximate versions.